pipeline.py

```diff
@@ -18,6 +20,7 @@
 JUDGE_SECRET = "judge-server"
 MOCKED_STAGES = ["sdg", "train", "eval"]
 PIPELINE_FILE_NAME = "pipeline.yaml"
+IMAGE_PULL_SECRET = "redhat-et-ilab-botty-pull-secret"
```

Comment:
Do we want to give this a more generic name?

Reply:
we will in a follow-up